docs/appendix/track.md

```diff
@@ -17,18 +17,20 @@ One of the most-often used traces is to find the origins of health problems like
 Here is the logic for tracking and tracing.
```

Comment:
Here is the logic finding the flow element that comes after a particular element (tracking) and before a particular element (tracing):